### DISCUSSION TITLE
C-Juno-106 Editor

### DISCUSSION THREAD

**Discussion by dnaldoog** [RomanKubiak/ctrlr#471] [Releases]
Here is an editor for the Roland Juno 106.

The GUI is based on the editor created by Roman Kubiak.

download here https://sourceforge.net/projects/c-juno-106/

I don't own a Roland Juno 106, so if there is anyone who does have one and can test the panel, that would be great.

![ss2](https://user-images.githubusercontent.com/76831683/171830982-098d5f9f-270d-4cdf-b465-5a994beb54c2.png)

**Comment by synthmania67** [comment]
Hi, I own a Juno 106 and will give your editor a try.

**Reply by dnaldoog** [reply]
Oh thank you!
 I have no idea whether it works or not. It’s the first panel I’ve done for a synth I don’t own. The panel started from trying to help someone on this forum with coding.
